tests/enum.rs

```diff
@@ -200,3 +204,34 @@ fn test_try_decode() {
     let e: WideEnum = val.try_into().expect("convert");
     assert_eq!(e, WideEnum::E);
 }
+
+#[derive(Encode, Decode, PartialEq, Debug)]
+pub enum GlColor {
+    #[fluvio_kf(tag = 1)]
+    GlTextureRedType = 0x8C10,
+    #[fluvio_kf(tag = 0)]
+    GlTextureGreenType = 0x8C11,
+    #[fluvio_kf(tag = 2)]
+    GlTextureBlueType = 0x8C12,
+}
+
+impl Default for GlColor {
+    fn default() -> Self {
+        Self::GlTextureRedType
+    }
+}
+
```

Comment:
can you add test for encode_discriminant?

Reply:
Sure